src/v/datalake/tests/parquet_writer_test.cc

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2024 Redpanda Data, Inc.
+ *
+ * Licensed as a Redpanda Enterprise file under the Redpanda Community
+ * License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://github.com/redpanda-data/redpanda/blob/master/licenses/rcl.md
+ */
+
+#include "datalake/arrow_translator.h"
+#include "datalake/parquet_writer.h"
+#include "datalake/tests/test_data.h"
+#include "iceberg/tests/value_generator.h"
+#include "utils/file_io.h"
+
+#include <arrow/array.h>
+#include <arrow/io/interfaces.h>
+#include <arrow/io/memory.h>
+#include <arrow/table.h>
+#include <arrow/type_fwd.h>
+#include <gtest/gtest.h>
+#include <parquet/arrow/reader.h>
+#include <parquet/type_fwd.h>
+
+TEST(ParquetWriter, DoesNothing) {
+    datalake::arrow_translator schema_translator(
+      test_schema(iceberg::field_required::no));
+
+    iobuf full_result;
+
+    for (int i = 0; i < 5; i++) {
+        auto data = iceberg::tests::make_value(
+          iceberg::tests::value_spec{
+            .forced_fixed_val = iobuf::from("Hello world")},
+          test_schema(iceberg::field_required::no));
+        schema_translator.add_data(std::move(data));
+    }
+
+    std::shared_ptr<arrow::Array> result = schema_translator.take_chunk();
+    ASSERT_NE(result, nullptr);
+
+    datalake::arrow_to_iobuf writer(*schema_translator.build_arrow_schema());
+
+    // The first write is a special case because it is 4 bytes longer.
```

Comment:
Curious, is this explaining why we're using EXPECT_NEAR instead of EXPECT_EQ? Generally wondering where the 4 is showing up in this test?